uv.lock

```diff
@@ -1,24 +1,16 @@
 version = 1
 requires-python = ">=3.10, <3.13"
+resolution-markers = [
+    "python_full_version < '3.13'",
```

Comment:
This looks odd, given `requires-python` explicitly excludes 3.13 (`<3.13` not `<=3.13`) ??

Reply:
I was confused about it, but it seems okay according to https://github.com/astral-sh/uv/issues/6795